common/js/src/i18n.js

```diff
@@ -94,22 +97,38 @@ function applyAriaLabelTranslation(element, translatedText) {
   element.setAttribute('aria-label', translatedText);
 }
 
+/**
+ * @param {!Element} element 
+ * @param {string} translatedText 
+ */
+function applyTitleTranslation(element, translatedText) {
+  logger.fine('Translating element.title [' + element.outerHTML +
+              '] to "' + translatedText + '"');
+  element.setAttribute('title', translatedText);
+}
+
 /** 
  * Takes the element passed, replacing element.textContent with 
- * translation if it contains I18N_DATA_ATTRIBUTE, and setting aria-label to 
- * translation if it contains I18N_DATA_ARIA_LABEL_ATTRIBUTE.
+ * translation if it contains I18N_DATA_ATTRIBUTE, setting aria-label to 
+ * translation if it contains I18N_DATA_ARIA_LABEL_ATTRIBUTE, and setting
+ * title to translation if it contains I18N_TITLE_ATTRIBUTE.
  * @param {!Element} element
  */
 GSC.I18n.adjustElementTranslation = function(element) {
   if (element.hasAttribute(I18N_DATA_ATTRIBUTE)) {
     transformElement(
-        element, I18N_DATA_ATTRIBUTE, applyTranslation);
+        element, I18N_DATA_ATTRIBUTE, applyContentTranslation);
   }
 
   if (element.hasAttribute(I18N_DATA_ARIA_LABEL_ATTRIBUTE)) {
     transformElement(
         element, I18N_DATA_ARIA_LABEL_ATTRIBUTE, applyAriaLabelTranslation);
   }
+
+  if (element.hasAttribute(I18N_TITLE_ATTRIBUTE)) {
```

Comment:
Could you please add this new attribute into the test? (would be great to do this in the same CL)